src/System.Text.Primitives/System/Text/Parsing/Unsigned.cs

```diff
@@ -0,0 +1,301 @@
+﻿
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+// NOTE: This file is generated via a T4 template. Please do not edit this file directly. Any changes should be made
+// in Unsigned.tt.
+
+namespace System.Text
+{
+    public static partial class PrimitiveParser
+    {
+        public static bool TryParseByte(ReadOnlySpan<byte> text, out byte value, out int bytesConsumed, EncodingData encoding = default(EncodingData), TextFormat format = default(TextFormat))
+        {
```

Comment:
Built-in invariant encoding does not work yet. No parsing trie.

        public EncodingData(byte[][] symbols, TextEncoder encoder)
        {
            _symbols = symbols;
            _encoder = encoder;
            _parsingTrie = null;
            _parsingTrie = CreateParsingTire(_symbols);
        }

        private ParsingTrieNode[] CreateParsingTire(byte[][] _digitsAndSymbols)
        {
            // TODO: this needs to be implemented;
            return null;
        }